client/views/msg_condensed.tpl

```diff
@@ -0,0 +1,7 @@
+<div class="msg {{type}} closed" data-time="{{time}}">
+	<span class="time hide-text">{{tz time}}</span>
+	<span class="from"><span class="expander"></span></span>
```

Comment:
> Are you doing this for users or just for yourself?

You know, I am just about done with you and your insidious comments and personal attacks, ALWAYS (not even talking about the comment where you talk about my self-esteem, who do you think you are?!).

You *clearly* have no idea what UX means, and what are the best practices in that regard. Your opinion is not fact. It is not the undisputed golden rule, it is not what we should blindly take. No, when you open a PR with 3 changes and the team disagrees with 2 of them, we *will not* merge it as is just because you say so.
I am more than happy to discuss any decision I make, and open to genuine and justified arguments. I suggest you start to do something similar.

I will also stop answering to you because I have noticed that you *constantly* quote people out of context to your advantage, and put words in people's mouth (I could find some examples, but I won't).
For the n-th time, I allocate a **tremendous** amount of my free, personal time, almost *every day* that I take off other things like hobbies, family, and sleep. It's a trade-off I am happy to make because mot people are very thankful for the work the entire team puts in, and it's very rewarding to know that this efforts are helping people and making their daily lives easier. But, hear me out, **I do not work for you**. You might have missed the many times I have said this, but I am genuinely thankful for the contributions you have made. It does not mean however that you are entitled to see all your PRs merged with no questions asked. I am not thankful for the tons of hours wasted on pointless arguments, and frankly, being shat upon by you.
I do not have the time, the energy, nor the will to deal with this kind of things.

One last thing: I sincerely hope that you'll realize the damages that you do, apologize for your rudeness, and start acting more respectfully of people and their work. I would be more than happy to restart with a clean slate, help you thrive in the open source world and, overall, make The Lounge an even better IRC client, as we all want. Based on previous exchanges, I do think however that you'll see me as the unthankful bad guy, which is a shame. My door will always be open for respectful and open-minded discussions.

*To the uninformed reader: this is not me taking out on a commenter out of the blue. There have been many of those interactions, to the point where frustration becomes too big to ignore. All team members have spoken up and asked for a better conduct before. I personally believe very strongly in open source and its values, and I don't think things should go that way. There is always room for healthy discussions, even when disagreeing, and I hope we can get back to that soon enough.*